test/unit/styling-cross-scope-apply.html

```diff
@@ -811,6 +876,35 @@
     CustomElements.takeRecords();
     assertComputed(e, '20px');
   });
+
+  test('Mixins can set "inherit" for a property', function() {
+    if (navigator.userAgent.match(/Trident/)) {
+      this.skip();
+    }
+    var e = document.createElement('x-inherit');
+    document.body.appendChild(e);
+    CustomElements.takeRecords();
+    assertComputed(e.$.child, 'rgb(0, 255, 0)', 'color');
+    if (!Polymer.Settings.useNativeCSSProperties) {
+      assertComputed(e.$.child, '2px');
+    } else {
+      // this is not supported for the apply shim
```

Comment:
Let's be more explicit so we remember this: "Setting `inherit` on a non-inheriting property is not supported under the apply shim."